templates/errors/submission-failed.html

```diff
@@ -7,13 +7,13 @@
 {% block main %}
 <h1>{{ _("Sorry, there is a problem") }}</h1>
 <p data-qa="retry">{{ _("You can try to <a href='{url}'>submit your survey again</a>").format(url=url_for('questionnaire.get_questionnaire')) }}</p>
-  {% if survey_type and (survey_type in SURVEY_TYPES_BUSINESS or survey_type in SURVEY_TYPES_DEFAULT or survey_type in SURVEY_TYPES_SOCIAL) %}
+  {% if survey_type and (survey_type in SURVEY_TYPES_BUSINESS or survey_type in SURVEY_TYPES_DEFAULT or survey_type in SURVEY_TYPES_SOCIAL or survey_type in SURVEY_TYPES_HEALTH) %}
```

Comment:
Set it as var so it's something like `ALL_SURVEY_TYPES = SURVEY_TYPES_BUSINESS + SURVEY_TYPES_DEFAULT + SURVEY_TYPES_SOCIAL +SURVEY_TYPES_HEALTH` and then only do `{% if survey_type and survey_type in ALL_SURVEY_TYPES %}`.